src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js

```diff
@@ -69,12 +69,12 @@
 
         const tooltipNodes = [...baseElement.querySelectorAll(TOOLTIPS_SELECTOR)];
 
-        if (baseElement instanceof HTMLElement) {
+        if (baseElement instanceof Element) {
             tooltipNodes.push(baseElement);
         }
 
         for (tooltipNode of tooltipNodes) {
-            if (tooltipNode.title) {
+            if (tooltipNode.hasAttribute('title')) {
```

Comment:
`HTMLElement` has `title` property, but not `SVGElement` even though it can have `title` attribute, thus we are checking we are using `hasAttribute` function.

Ref:
- https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/title
- https://developer.mozilla.org/en-US/docs/Web/API/SVGElement